gui/RespiraConnectedDevice.h

```diff
@@ -0,0 +1,115 @@
+#include "../common/generated_libs/network_protocol/network_protocol.pb.h"
+#include "../common/third_party/nanopb/pb_common.h"
+#include "../common/third_party/nanopb/pb_decode.h"
+#include "../common/third_party/nanopb/pb_encode.h"
+#include "connected_device.h"
+#include <QSerialPort>
+#include <QThread>
+
+#define INTER_FRAME_TIMEOUT 11
+
+class RespiraConnectedDevice : public ConnectedDevice {
```

Comment:
Please document what this does and mention the thread safety considerations: specifically, for the moment, Send and Receive have to be called from the same thread.

Reply:
Done